index.js

```diff
@@ -80,10 +88,17 @@ async function pay (plugin, {
 
     const payStream = ilpConn.createStream()
 
-    return Promise.race([
-      payStream.sendTotal(sendAmount).then(() => payStream.end()),
-      new Promise(resolve => payStream.on('end', resolve))
-    ])
+    try {
+      await payStream.sendTotal(sendAmount, { timeout: streamOpts.timeout })
+    } catch (err) {
+      const totalSent = payStream.totalSent
+      await ilpConn.end()
```

Comment:
We have the same problem here that we have in the pull case. If the timeout is too high, ilpConn.end() will throw the btp error. In this case, the PaymentError is not thrown and the CLI does not know what `totalSent` is and just says it sent 0. 
I think we should also wrap this into a try-catch and log the error using ilp-logger.

Reply:
Thanks. I'll wait for https://github.com/interledgerjs/ilp-protocol-spsp/pull/4 to be merged and then fix to avoid conflicts.